src/types/register/register.ts

```diff
@@ -0,0 +1,5 @@
+export type RegisterUserData = {
```

Comment:
If you open API spec you'll see that the same entity has the name User. My suggestion is to use names for UI model from API. It'll help us to work with entities easier

Reply:
Vlad, the name of model in spec is SignUpCred. And it's really credentials not an user. I think SignUpCredentials or for instance RegistrationUserData is suitable names but not a User.
![image](https://user-images.githubusercontent.com/40975346/152462542-5ead4e19-6dc7-4d0f-aac0-8ed062f1a53e.png)